scripts/qa_crowbarsetup.sh

```diff
@@ -923,7 +923,7 @@ function onadmin_prepare_cloud_repos
             addcloud6pool
             addcloud6maintupdates
             ;;
-        develcloud7|susecloud7|M?|Beta*|RC*|GMC*)
+        develcloud7|susecloud7|GM7|GM7+up|M?|Beta*|RC*|GMC*)
             addcloud7pool
             addcloud7maintupdates
```

Comment:
when looking above, the difference between GM and GM+up is that the latter has maintenance update repos - we should keep that distinction here, too